editor/src/components/editor/editor-component.tsx

```diff
@@ -677,12 +679,19 @@ const LockedOverlay = React.memo(() => {
 
   const editorLocked = React.useMemo(() => githubOperations.length > 0, [githubOperations])
 
-  const refreshingDependencies = useEditorState(
+  const refreshingDependencies = false
+  useEditorState(
```

Comment:
The result of the `useEditorState` call is discarded here, was this an intentional change?

Reply:
no, it's an error! (should have been assigned to `refreshingDependencies` instead of `false` which I only used temporarily before the feature switch. Thank you!